articles/azure-sql/virtual-machines/windows/performance-guidelines-best-practices-checklist.md

```diff
@@ -139,8 +139,10 @@ For your SQL Server availability group or failover cluster instance, consider th
 * If optimizing SQL Server VM performance does not resolve your unexpected failovers, consider [relaxing the monitoring](hadr-cluster-best-practices.md#relaxed-monitoring) for the availability group or failover cluster instance. However, doing so may not address the underlying source of the issue and could mask symptoms by reducing the likelihood of failure. You may still need to investigate and address the underlying root cause. For Windows Server 2012 or higher, use the following recommended values: 
    - **Lease timeout**: Use this equation to calculate the maximum lease time out value:   
     `Lease timeout < (2 * SameSubnetThreshold * SameSubnetDelay)`.    
-    Start with 40 seconds. If you're using the relaxed `SameSubnetThreshold` and `SameSubnetDelay` values recommended previously, do not exceed 80 seconds for the lease timeout value.    
-   - **Max failures in a specified period**: Set this value to 6. 
+    Start with 40 seconds. If you're using the relaxed `SameSubnetThreshold` and `SameSubnetDelay` values recommended previously, do not exceed 80 seconds for the lease timeout value. 
+   - **Max failures in a specified period**: Set this value to 6.
+   - **Session timeout**: Set this value to 20.
```

Comment:
```suggestion
```

Reply:
Please remove session timeout, we don't want to specify a value here as it could have a negative impact on sync cycles